docs/source/how_tos/spark_cluster_on_kestrel.rst

```diff
@@ -39,7 +39,7 @@ Steps
 
 .. code-block:: console
 
-    $ configure_and_start_spark.sh -c /projects/dsgrid/containers/spark350_py311.sif
+    $ configure_and_start_spark.sh -c /datasets/images/apache_spark/spark351_py312.sif
```

Comment:
Should this also include -M 10?

Reply:
Sure. This setting is not so clear-cut, though. It's my best guess for now.